widgetastic_manageiq/__init__.py

```diff
@@ -2666,43 +2689,54 @@ class NonJSBaseEntity(View):
     list_entity = BaseListEntity
     tile_entity = BaseTileIconEntity
 
-    def __init__(self, parent, name, logger=None):
+    def __init__(self, parent, id, logger=None):
         View.__init__(self, parent, logger=logger)
-        self.name = name
+        self.id = id
 
     def _get_existing_entity(self):
         for item in (self.quad_entity, self.tile_entity, self.list_entity):
-            if item(name=self.name).is_displayed:
-                return item(name=self.name)
+            if item(id=self.id).is_displayed:
+                return item(id=self.id)
         else:
-            raise NoSuchElementException("Item {name} isn't found on page".format(name=self.name))
+            raise NoSuchElementException("Item {id} isn't found on page".format(id=self.id))
 
     def __getattr__(self, name):
-        if name.startswith('__'):
+        if name.startswith('__') or name == 'is_displayed':
```

Comment:
that bit looks wrong - is_displayed is a property, getattr shouldn't even care

Reply:
you're right. my bad, I forgot to remove it